OrbitCore/LibunwindstackUnwinder.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <asm/perf_regs.h>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "unwindstack/MachineX86_64.h"
+#include "unwindstack/RegsX86_64.h"
+#include "unwindstack/Unwinder.h"
+
+class LibunwindstackUnwinder {
+ public:
+  bool SetMaps(const std::string& maps_buffer);
+
+  [[nodiscard]] std::vector<unwindstack::FrameData> Unwind(
```

Comment:
I'm not sure I would mark this as "nodiscard". Could there be use cases for this function that don't want to use the return value? Also, not using the return value doesn't really cause bad failures (as opposed to not checking an error code return value).